packages/react-conformance/src/utils/getCallbackArguments.test.ts

```diff
@@ -34,10 +34,11 @@ function createCompilerHost(): ts.CompilerHost {
     ...originalHost,
 
     readFile: () => {
-      throw new Error('Not supported');
+      //   throw new Error('Not supported');
```

Comment:
@layershifter do you remember why you set it up like this ? with ts bump suddenly this paths started to get executed https://github.com/microsoft/fluentui/pull/19369

Reply:
@Hotell no, I cannot recall 🤔

Reply:
hmm ok I'll leave it as is for now. 

note for myself: this test is extremely slow - 20 seconds on my local machine ( m1 max ) so on CI it will take 3 times more - `68.34s`